### ISSUE TITLE
can you provide the loss tendencies？

### ISSUE THREAD

**Issue by Sierkinhane** [NiteshBharadwaj/part-affinity#5]
I want to know the train loss of your training

**Comment by ericzw** [comment]
> I want to know the train loss of your training

hey, could you show me your training loss curve?
